doc/SPIRE101.md

```diff
@@ -92,7 +92,7 @@ server {
 }
 
 plugins {
-        ControlPlaneCA "ca" {
+        ServerCA "ca" {
```

Comment:
Ahh this is out-of-date... should be `ServerCA "memory"`. Can also remove all the references to `plugin_cmd` and `plugin_checksum`